classes/PaymentModule.php

```diff
@@ -819,6 +819,14 @@ public function validateOrder($id_cart, $id_order_state, $amount_paid, $payment_
                 PrestaShopLogger::addLog('PaymentModule::validateOrder - End of validateOrder', 1, null, 'Cart', (int)$id_cart, true);
             }
 
+            Hook::exec('actionValidateOrderAfter', [
+                'cart' => $this->context->cart,
+                'order' => $order,
```

Comment:
Why not pass only the order object? Other params are from Context and from the order object itself.